app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,35 @@
+class RelationshipsController < ApplicationController
+  before_action :check_logged_in, only: %i[create destroy]
+
+  # POST /users/:user_id/relationships
+  def create
+    follow = current_user.active_relationships.build(follower_id: params[:user_id])
+    if follow.save
+      flash[:notice] = 'フォローしました'
+      redirect_to users_path
+    else
+      flash.now[:alert] = 'フォローに失敗しました'
+      render users_path
+    end
+  end
+
+  # DELETE /users/:user_id/relationships
+  def destroy
+    follow = find_follow(params[:user_id])
```

Comment:
IMO

メソッドに切り分けるほどでもないと思う！
```suggestion
    follow = \
      current_user.active_relationships.find_by!(follower_id: params[:user_id])
```

Reply:
rubocopの`Metrics/AbcSize`に引っかかったからメソッド切り分けたけど
逆にわかりづらくなってた気がしてた...
メソッド元に戻してrubocop_todo.ymlで制限緩くしました。
[a7aa4eb](https://github.com/ok-os-job-change-team/kaito-twitter-clone-bootcamp/pull/57/commits/a7aa4eb1560c611aaaf7f8d89b06910fd694a357)

Reply:
rubocop_todo.ymlで制限を緩くするのはやめよう